### PR TITLE
Release v0.13.0

### PR DESCRIPTION
**Pull request by romac**
Closes: #1992

## Guide updates

- [x] https://github.com/informalsystems/ibc-rs/issues/1927 (@romac)

### CODE REVIEW

File: guide/src/config.md [hunk 1]
```diff
@@ -48,70 +48,48 @@ please refer to the [Keys](./commands/keys/index.md) sections in order to learn
...
+There are two kinds of ICA channels:
+
+1. The host channels, whose port is `icahost`
+2. The controller channels, whose port starts with `icacontroller-` followed
+   by the owner account address. [See the spec for more details][ica].
+
+If you wish to only relay on a few specific standard channels (here `channel-0` and `channel-1`),
+but also relay on all ICA channels, you can specify the following packet filter:
+
+> Note the use of wildcards in the port and channel identifiers
```
Comment (adizere) [inline guide/src/config.md:70]:
```suggestion
> Note the use of wildcards in the port and channel identifiers (`['ica*', '*']`)
```

**Review by adizere — APPROVED** [review]
Pre-approved, left just a couple minor suggestions!

File: .changelog/config.toml [hunk 2]
```diff
@@ -71,8 +71,9 @@ entry_indent = 2
...
-    ibc             = { name = "IBC Modules",     path = "modules" }
-    ibc-proto       = { name = "IBC Proto",       path = "proto" }
-    ibc-relayer     = { name = "Relayer Library", path = "relayer" }
-    ibc-relayer-cli = { name = "Relayer CLI",     path = "relayer-cli" }
-    guide           = { name = "Guide",           path = "guide" }
+    ibc                  = { name = "IBC Modules",                path = "modules" }
+    ibc-proto            = { name = "IBC Proto",                  path = "proto" }
+    ibc-relayer          = { name = "Relayer Library",            path = "relayer" }
+    ibc-relayer-cli      = { name = "Relayer CLI",                path = "relayer-cli" }
+    ibc-integration-test = { name = "Integration Test Framework", path = "relayer-cli" }
```
[the remark this answers is not in the document]
Comment (romac) [reply .changelog/config.toml:78]: Fixed, thanks!

File: .changelog/v0.13.0/summary.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+This release also add support for wildcards in port and channel identifiers
+in the packet filter configuration, which enable operators to filter
+ICA channels based on the port prefix.
+
+Additionally, the IBC Protocol Buffers definitions can now be used from CosmWasm.
+
+## Note for operators
+
+To enable or disable relaying on [Interchain Accounts][ica] channels,
+please check out the [new section in the guide][guide-ica].
```
[the remark this answers is not in the document]
Comment (romac) [reply .changelog/v0.13.0/summary.md]: Done!

**Pull request merged**